CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [UNRELEASED]
 
```

Comment:
Can we add a `### Breaking changes` section with a bullet on how `shiny.run()`'s app,host,port are positional args and everything else is now a keyword arg?